spec/features/merchants/merchants_index_page_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "All users can see a merchants index page", type: :feature do
+  before :each do
+    @merchant_1 = create(:merchant)
+    @merchant_2 = create(:merchant)
+    @merchant_3 = create(:inactive_merchant)
+  end
+
+  context 'as any kind of user on the system' do
+    it 'shows all merchants in the system except disabled ones' do
+
+      visit merchants_path
+
+      within "#merchant-#{@merchant_1.id}" do
+        expect(page).to have_link(@merchant_1.name, href: merchant_path(@merchant_1))
+        expect(page).to have_content(@merchant_1.city)
+        expect(page).to have_content(@merchant_1.state)
+        expect(page).to have_content(@merchant_1.created_at)
+      end
+
+      within "#merchant-#{@merchant_2.id}" do
+        expect(page).to have_link(@merchant_2.name, href: merchant_path(@merchant_2))
+        expect(page).to have_content(@merchant_2.city)
+        expect(page).to have_content(@merchant_2.state)
+        expect(page).to have_content(@merchant_2.created_at)
+      end
+
+      expect(page).to_not have_link(@merchant_3.name, href: user_path(@merchant_3))
+      expect(page).to have_content(@merchant_3.created_at)
```

Comment:
I think you meant this line to be a `.to_not`
Its failing on my machine locally though because its the same created_at as merchant_1 and 2.  Maybe just test to not have the name, city, state instead.

Reply:
Those can all be repeated just like the timestamp. So I just changed it to_not have the user id in the css, which is a unique attribute.